packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/index.js

```diff
@@ -13,14 +14,16 @@ import ExternalResource from './ExternalResource';
 
 import { ErrorMessage } from './elements';
 
-function Dependencies({ signals = null, store = null }) {
-  const sandbox = store.editor.currentSandbox;
+const Dependencies = () => {
+  const { editor } = useStore();
```

Comment:
@SaraVieira You can even destructur `editor` a bit further

```suggestion
  const { editor: { currentSandbox: sandbox, parsedConfigurations: { package } } } = useStore();
```

Reply:
I didn't do it like that because in my and ives idea makes it more confusing